core/nodejsActionBase/launcher.js

```diff
@@ -27,9 +29,8 @@ try {
     } catch (e) {}
   })();
 
-  const lambdaCompat = process.env.__OW_LAMBDA_COMPAT === undefined ? false : process.env.__OW_LAMBDA_COMPAT.toLowerCase() === 'true' && NodeActionLambdaRunner !== undefined;
   const handler = eval('require("./##MAIN_FILE##").##MAIN_FUNC##') // Will be replaced in the compile script with the correct main.
-  const runner = lambdaCompat === false ? new NodeActionRunner(handler) : new NodeActionLambdaRunner(handler);
+  const runner = !isLambda(handler) ? new NodeActionRunner(handler) : new NodeActionLambdaRunner(handler);
```

Comment:
As mentioned in the PR against main, we probably have to keep backwards compatibility here in that the existing lambda runtimes have to continue to work. I'd propose to still use the __OW_LAMBDA_COMPAT variable here and check if either it is set or if the passed handler looks like a lambda handler.

Reply:
I changed the runtime to be backwards compatible like this. See latest squashed commit.